easybuild/easyconfigs/e/EvidentialGene/EvidentialGene-2018.01.01-intel-2017a-Perl-5.24.1.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = "PackedBinary"
+
+name = "EvidentialGene"
+version = '2018.01.01'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://arthropods.eugenes.org/EvidentialGene/'
+description = """EvidentialGene is a genome informatics project for
+ "Evidence Directed Gene Construction for Eukaryotes",
+ for constructing high quality, accurate gene sets for
+ animals and plants (any eukaryotes), being developed by
+ Don Gilbert at Indiana University, gilbertd at indiana edu."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = [
+    'http://arthropods.eugenes.org/EvidentialGene/other/evigene_old/',
+    'http://arthropods.eugenes.org/EvidentialGene/other/evigene_old/evigene_older/',
+]
+month = ['', 'jan', 'feb', 'mar', 'apr', 'may', 'jun', 'jul', 'aug', 'sep', 'oct', 'nov', 'dec']
+dlver = '%(version_major)s'[-2:] + month[int(version.split('.')[1])] + version.split('.')[2]
```

Comment:
@hajgato The `'%(version_major)s'[-2:]` prematurely resolves to `)s`, which doesn't work either...

You'll need to replace it with `version.split('.')[0][-2:]`?